SYS_ATL/API.py

```diff
@@ -1031,6 +1031,22 @@ def par_to_seq(self, par_pattern):
 
         return Procedure(loopir, _provenance_eq_Procedure=self)
 
+    def lift_alloc_simple(self, alloc_site_pattern, n_lifts=1):
+        if not is_pos_int(n_lifts):
+            raise TypeError("expected second argument 'n_lifts' to be "
+                            "a positive integer")
+
+        stmts_len = len(self._find_stmt(alloc_site_pattern, default_match_no=None))
+        loopir = self._loopir_proc
+        for i in range(0, stmts_len):
+            s = self._find_stmt(alloc_site_pattern, body=loopir.body, default_match_no=None)[i]
```

Comment:
I find this loop construction very confusing and I'm not 100% sure it's correct... at the very least there should be some test that lifts multiple allocations with this directive.